### ISSUE TITLE
Make it Responsive.

### ISSUE THREAD

**Issue by JenilGajjar20** [JenilGajjar20/food-app#1]
You have to make the site responsive using Tailwind CSS. Should have knowledge of [BEM(Block, Element, Modifier)](https://getbem.com/) methodology. Changes should be done only in the CSS files.

**Comment by TandinDev** [comment]
Hello @JenilGajjar20 can i work on this?

**Comment by JenilGajjar20** [comment]
Hey, @TandinDev. Yes, you can. Are you contributing as a part of **Hacktoberfest** ?

**Comment by Ankitmohanty2** [comment]
is this issue solved? i can contribute

**Comment by TandinDev** [comment]
@JenilGajjar20 I tried to run project but it gave me mongo DB issue, is there env I need to create?

**Comment by JenilGajjar20** [comment]
> @JenilGajjar20 I tried to run project but it gave me mongo DB issue, is there env I need to create?

@TandinDev Yes you need to create an env file. Setup the mongo_uri and cookie_secret.

**Comment by JenilGajjar20** [comment]
> is this issue solved? i can contribute

Hey, @Ankitmohanty2. Yes, you can contribute to this issue as well.

**Comment by Ankitmohanty2** [comment]
ok thank you

**Comment by TandinDev** [comment]
@JenilGajjar20 do you have mongo_uri and cookie_secret? Because I am not that fimilar with this.

**Comment by TandinDev** [comment]
Hello @JenilGajjar20 I am sorry, I couldn't make it work.
